scripts/kani-regression.sh

```diff
@@ -5,6 +5,25 @@
 # To enable "unsound_experimental features, run as follows:
 # `KANI_ENABLE_UNSOUND_EXPERIMENTS=1 scripts/kani-regression.sh`
 
+# Check if kissat has the minimum required version
```

Comment:
can you move this to its own check_kissat_version? Or even augment the CBMC to include the kissat version.

Reply:
I moved it to a separate script. Didn't feel it should be part of the CBMC one. Perhaps as a cleanup, we can create a single script, e.g. `check_dep_versions` that calls the other scripts.